payloads/v9/_interactions/slashCommands.ts

```diff
@@ -0,0 +1,265 @@
+import { APIRole, APIUser } from '../index';
+import { Permissions, Snowflake } from '../../../globals';
+import { APIPartialChannel } from '../channel';
+import { APIGuildMember } from '../guild';
+import { APIBaseInteraction } from './base';
+import { InteractionType } from './responses';
+
+/**
+ * https://discord.com/developers/docs/interactions/slash-commands#applicationcommand
+ */
+export interface APIApplicationCommand {
+	/**
+	 * Unique id of the command
+	 */
+	id: Snowflake;
+	/**
+	 * Unique id of the parent application
+	 */
+	application_id: Snowflake;
```

Comment:
```suggestion
	application_id: Snowflake;
	/**
	 * Guild id of the command, if not global
	 */
	guild_id?: Snowflake;
```
Unsure if this belongs here but it was missing